src/subcommands/deployments.ts

```diff
@@ -401,11 +473,246 @@ function renderOverview(
+            : `https://${project.name}-${build.deploymentId}.${ingressRoot}.dev`,
+        ),
+        Entrypoint: colorByStatus(deploymentEntrypoint(build)),
+        ...build.relatedCommit
+          ? {
+            Branch: colorByStatus(build.relatedCommit.branch),
+            Commit: colorByStatus(build.relatedCommit.hash.slice(0, 7)),
+          }
+          : {},
+      };
+      return row;
+    });
+    renderTable(table);
+
+    if (paging.page + 1 >= paging.totalPages) {
+      return;
+    }
+    alert(`Press enter to fetch the next page`);
+    tty.goUpSync(1, Deno.stdout);
+    tty.clearDownSync(Deno.stdout);
+    const nextPage = paging.page + 1;
+    const spinner = wait(
+      `Fetching page ${
+        // API page param is 0-based
+        nextPage +
+        1} of the list of deployments of project '${project.name}'...`,
+    )
+      .start();
+    const deploymentsNextPage = await api.listDeployments(
+      project.id,
+      nextPage,
+      paging.limit,
+    );
+    if (!deploymentsNextPage) {
+      spinner.fail(
+        `The project '${project.name}' does not exist, or you don't have access to it`,
+      );
+      return Deno.exit(1);
+    }
+    deployments = deploymentsNextPage[0];
+    paging = deploymentsNextPage[1];
+    spinner.succeed(
+      `Page ${
+        paging.page + 1
+      } of the list of deployments of the project '${project.name}' is ready`,
+    );
+  }
+}
+
+function isCurrentProd(project: Project, build: Build): boolean {
+  return project.productionDeployment?.id === build.id;
+}
+
+function deploymentError(build: Build): DeploymentProgressError | undefined {
+  return build.logs.find((log): log is DeploymentProgressError =>
+    log.type === "error"
+  );
+}
+
+function deploymentStatus(
+  project: Project,
+  build: Build,
+): DeploymentStatus {
+  const isError = deploymentError(build) !== undefined;
+  const isPending = !isError &&
+    (build.deployment === null ||
+      build.deployment.domainMappings.length === 0);
+  return isError
+    ? "Failed"
+    : isPending
+    ? "Pending"
+    : isCurrentProd(project, build)
+    ? "Production"
+    : "Preview";
+}
+
+function isReady(status: DeploymentStatus): boolean {
+  return ["Production", "Preview"].includes(status);
+}
+
+type DeploymentStatus = "Failed" | "Pending" | "Production" | "Preview";
+
+function deploymentDatabase(
+  databases: Database[],
+  build: Build,
+): Database | undefined {
+  return databases.find((db) =>
+    db.databaseId === build.deployment?.kvDatabases["default"]
+  );
+}
+
+function deploymentLocaleDate(build: Build): string {
+  return new Date(build.createdAt).toLocaleString(navigator.language, {
+    timeZoneName: "short",
+  });
+}
+
+function deploymentRelativeDate(build: Build): string {
+  const createdAt = new Date(build.createdAt);
+  const sinces = [new Date().getTime() - createdAt.getTime()];
+  const sinceUnits = ["milliseconds ago"];
+  if (sinces[0] >= 1000) {
+    // Remove milis
```

Comment:
nit - "millis" (double L) is more widely used I guess?
```suggestion
    // Remove millis
```